src/main/scala/com/gu/newsletterlistcleanse/UpdateBrazeUsersLambda.scala

```diff
@@ -23,47 +25,53 @@ object UpdateBrazeUsersLambda {
 
   val timeout: Duration = Duration(15, TimeUnit.MINUTES)
 
-  val brazeClient: BrazeClient = new BrazeClient()
-
-
   def handler(sqsEvent: SQSEvent): Unit = {
     val env = Env()
     logger.info(s"Starting $env")
     parseCleanseListSqsMessage(sqsEvent) match {
       case Right(cleanseLists) =>
         process(cleanseLists)
-      case Left(parseError) =>
-        logger.error(parseError.getMessage)
+      case Left(parseErrors) =>
+        parseErrors.foreach(e => logger.error(e.getMessage))
     }
 
   }
 
-  def parseCleanseListSqsMessage(sqsEvent: SQSEvent): Either[circe.Error, List[CleanseList]] = {
+  def parseCleanseListSqsMessage(sqsEvent: SQSEvent): Either[List[circe.Error], List[CleanseList]] = {
     (for {
       message <- sqsEvent.getRecords.asScala.toList
     } yield {
       decode[CleanseList](message.getBody)
     }).toEitherList
   }
 
-  def process(cleanseLists: List[CleanseList]): Either[Error, List[HttpResponse[String]]] = {
+  def process(cleanseLists: List[CleanseList]): Either[List[BrazeError], List[BrazeResponse]] = {
 
-    for {
+    val brazeResponses = for {
       cleanseList <- cleanseLists
       newsletterName = cleanseList.newsletterName
       userIds = cleanseList.userIdList
-    } {
-      ???
-      // TODO: Create API request
+      userId <- userIds
+      identityNewsletter <- getIdentityNewsletterFromName(newsletterName)
+    } yield {
 
-      // TODO: Send API request
+      val apiKey: String = Option(System.getenv("BRAZE_API_KEY")).getOrElse("")
```

Comment:
We probably want to add a configuration library to our lambdas, but that's for another pull request. I'll create a card